packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -90,9 +90,11 @@ export async function createNewFluidFile(
                 }
                 event.end({
                     headers: Object.keys(headers).length !== 0 ? true : undefined,
+                    ...fetchResponse.commonSpoHeaders,
                 });
                 return content.itemId;
-            });
+            },
+            true);
```

Comment:
Would appreciate if you wait for and merge with https://github.com/microsoft/FluidFramework/pull/5363

Reply:
sure